src/plugins/workspace/public/plugin.ts

```diff
@@ -547,7 +557,17 @@ export class WorkspacePlugin
       });
     }
 
-    return {};
+    registerDefaultCollaboratorTypes({
+      getStartServices: core.getStartServices,
+      collaboratorTypesService: this.collaboratorTypes,
+    });
+
+    return {
+      setCollaboratorTypes: (types: WorkspaceCollaboratorType[]) => {
+        this.collaboratorTypes.setTypes(types);
+      },
+      getAddCollaboratorsModal: () => AddCollaboratorsModal,
```

Comment:
Shall we follow this pattern? It looks when plugin expose components, it exposes under `ui`  property
```suggestion
      ui: { AddCollaboratorsModal },
```